espresso/environment/2_espresso_liveness_test.go

```diff
@@ -2,15 +2,27 @@ package environment_test
 
 import (
 	"context"
```

Comment:
I needed to add this line to make the code compile (inside my IDE):
```
github.com/ethereum/go-ethereum/common
```

Reply:
Ooof, missed this import.

Fixed in b6eefeb54d3131358fba1aa5502994c3eb0fa47a.